src/nvim/main.c

```diff
@@ -1808,7 +1823,7 @@ static void source_startup_scripts(mparm_T *parmp)
      * - second user exrc file ($VIM/.exrc for Dos)
      * The first that exists is used, the rest is ignored.
      */
-    if (process_env((char_u *)"VIMINIT", TRUE) != OK) {
+    if (process_env((char_u *)"NVIMINIT", TRUE) != OK) {
```

Comment:
If we don't support VIMINIT then we definitely should also remove support for EXINIT. VIMINIT exists only as an alternative to EXINIT for users who want separate Vi and Vim `INIT`s :)